spec/helpers/medications_helper_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+describe MedicationsHelper do
+  let(:current_user) { create(:user) }
+  let(:medication) { create(:medication, user: current_user) }
+
+  before(:each) do
+    @medication = medication
```

Comment:
Why are we converting this to an instance method?

Reply:
because in `medications_helper#common_fields` the medication variable is called as an instance variable. [Example](https://github.com/ifmeorg/ifme/blob/master/app/helpers/medications_helper.rb#L29).